File: examples/tentacles/hello-world/js/handler.js
const fs = require("fs");
const req = JSON.parse(fs.readFileSync(0, "utf8") || "{}");
const name = (req.payload && req.payload.name) || "world";
process.stdout.write(JSON.stringify({
  v: 1, id: req.id, ok: true,
  result: { arm: "hello-js", lang: "javascript", hello: `hello, ${name}` },
}));

File: examples/tentacles/hello-world/python/handler.py
#!/usr/bin/env python3
import json, sys

req = json.loads(sys.stdin.read() or "{}")
name = (req.get("payload") or {}).get("name", "world")
sys.stdout.write(json.dumps({
    "v": 1, "id": req.get("id"), "ok": True,
    "result": {"arm": "hello-py", "lang": "python", "hello": f"hello, {name}"},
}))
